### PR TITLE
Create the "Useful Resources" Page

### PR DESCRIPTION
**Pull request by egecaglar**
Created the "Useful Resources" page. I was not sure if adding more explanations would make it harder for users to find the links - if adding explanations make sense I can quickly add them!

### CODE REVIEW

File: Tutorials/useful_resources.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: Useful Resources
+parent: Tutorials
+nav_order: 10
+mathjax: true
+---
+**Last Updated:** 2021-05-10
```
Comment (romesco) [inline Tutorials/useful_resources.md:8]: Make sure you update the date!

File: Tutorials/useful_resources.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+🚧 Work In Progress
```
Comment (romesco) [inline Tutorials/useful_resources.md:10]: You can probably remove this tag.

File: Tutorials/useful_resources.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+#### Pytorch Ignite:
+https://github.com/pytorch/ignite
+#### Pytorch Lightning:
+https://github.com/PyTorchLightning/pytorch-lightning
+
+https://www.youtube.com/playlist?list=PLaMu-SDt_RB5NUm67hU2pdE75j6KaIOv2
+
+https://www.youtube.com/watch?v=w10WrRA-6uI
+#### Pytorch Lightning Bolts:
+https://lightning-bolts.readthedocs.io/en/latest/yuo
```
Comment (romesco) [inline Tutorials/useful_resources.md:88]: Need to see if these are actually maintained or not. But could be good.

**Review by romesco — CHANGES_REQUESTED** [review]
Looks good! Just check into a few of my comments and I'll merge =].

**Review by yxyang — APPROVED** [review]
lgtm

**Comment by fishbotics** [comment]
Sorry it took so long, just wanted to make sure I reviewed this carefully. LGTM

**Comment by romesco** [comment]
DENIED

File: Tutorials/useful_resources.md (hunk 3 above)
Comment (fishbotics) [reply Tutorials/useful_resources.md:88]: Turns out these were not maintained because the URLs are dead

Comment (fishbotics) [reply Tutorials/useful_resources.md:88]: https://lightning-bolts.readthedocs.io/en/latest/

**Pull request closed without merging**